04_condition_variable/exercises/03b_ping_pong_easier.cpp

```diff
@@ -39,8 +36,10 @@ class PingPong {
                 this_thread::sleep_for(500ms);
             }
 
-            else
+            else{
+                 unique_lock<mutex> l(m_);
                 printingFunction("Ping repetition limit");
+            }
```

Comment:
Co Ty na to, żeby to przenieść za while? Wtedy powyższego ifa też można się pozbyć. Tylko w while pewnie trzeba będzie zmienić <= na <
Po tym już powinno wyglądać dobrze. Dla ponga identycznie :)